src/main/java/org/folio/circulation/support/ItemRepository.java

```diff
@@ -113,6 +113,13 @@ private ItemRepository(
       location.getPrimaryServicePointId());
   }
 
+  public CompletableFuture<Response> updateItem(Item item) {
```

Comment:
Thank you for adding this method.

It would be useful if this replaced the direct usage of the itemStorageClient in the `UpdateItem` class, or at the very least was consistent with it (as the logic there involves mapping to a storage representation that goes beyond the inner JSON in the class).